src/Server/Features/FreeAgents/List.cs

```diff
@@ -77,10 +78,14 @@ public ListMappingProfile()
                 s.Bids.GetHighestBidder().TeamId == currentUserTeamId)
             )
             .ForMember(d => d.BiddingEnds, mo => mo.MapFrom(s => s.EndOfFreeAgency!.Value.ToShortDateString()))
+            .ForMember(d => d.BiddingEndDate, mo => mo.MapFrom(s => s.EndOfFreeAgency))
```

Comment:
I don’t think having two properties with different formatting is the way to go. Thoughts?

Reply:
Agree, I tried to remove them, I guess I am just flaking